com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/listeners/EnteredTestingPhaseListener.java

```diff
@@ -0,0 +1,24 @@
+package com.sap.cloud.lm.sl.cf.process.listeners;
+
+import com.sap.cloud.lm.sl.cf.process.Constants;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Named;
+
+@Named("enteredTestingPhaseListener")
+public class EnteredTestingPhaseListener extends AbstractProcessExecutionListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EnteredTestingPhaseListener.class);
+
+    @Override
+    protected void notifyInternal(DelegateExecution context)  {
+        context.setVariable(Constants.VAR_ENTERED_TESTING_PHASE, true);
+    }
```

Comment:
See `PrepareAppsRestartStep`. Can't we do something similar to what is done there?